--- origsrc/glade-3.10.0/gladeui/glade-utils.c	2011-04-04 05:10:31.000000000 -0500
+++ src/glade-3.10.0/gladeui/glade-utils.c	2011-05-24 21:44:40.776311500 -0500
@@ -983,7 +983,7 @@ glade_util_have_devhelp (void)
                 {
                   /* Devhelp 0.12 required.
                    */
-                  if (major >= 0 && minor >= 12)
+                  if (major >= 2 || (major == 0 && minor >= 12))
                     have_devhelp = 1;
                   else
                     g_message (DEVHELP_OLD_MESSAGE);
