https://bugzilla.gnome.org/show_bug.cgi?id=646997

--- origsrc/glade-3.10.0/doc/Makefile.am	2011-04-04 06:29:25.000000000 -0500
+++ src/glade-3.10.0/doc/Makefile.am	2011-04-07 03:07:19.855677200 -0500
@@ -11,6 +11,8 @@ AUTOMAKE_OPTIONS = 1.6
 # The name of the module, e.g. 'glib'.
 DOC_MODULE=gladeui
 
+DOC_MODULE_VERSION=2
+
 # The top-level SGML file. You can change this if you want to.
 DOC_MAIN_SGML_FILE=$(DOC_MODULE)-docs.sgml
 
--- origsrc/glade-3.10.0/src/main.c	2011-04-04 06:29:25.000000000 -0500
+++ src/glade-3.10.0/src/main.c	2011-05-24 22:19:21.675332100 -0500
@@ -150,7 +150,7 @@ main (int argc, char *argv[])
     }
 
   g_set_application_name (APPLICATION_NAME);
-  gtk_window_set_default_icon_name ("glade-3");
+  gtk_window_set_default_icon_name ("glade");
 
   glade_setup_log_handlers ();
 
